src/amo/components/UserProfileEdit/index.js

```diff
@@ -0,0 +1,364 @@
+                {i18n.gettext(`Tell users a bit more information about yourself.
+                  These fields are optional, but they'll help other users get to
+                  know you better.`)}
+              </p>
+
+              <label className="UserProfileEdit--label" htmlFor="displayName">
+                {i18n.gettext('Display Name')}
+              </label>
+              <input
+                className="UserProfileEdit-displayName"
+                disabled={!user}
+                id="displayName"
+                name="displayName"
+                ref={(ref) => { this.displayName = ref; }}
+                defaultValue={user && user.displayName}
+              />
+
+              {/*
+                TODO: Don't show these to users who don't have a public-facing
+                user profile page (eg are developers). It's just noise and may
+                encourage them to enter a lot of text (especially the bio) which
+                no one will see. It also gets in the way of settings,
+                like notifications, below.
+              */}
+              <label className="UserProfileEdit--label" htmlFor="homepage">
+                {i18n.gettext('Homepage')}
+              </label>
+              <input
+                className="UserProfileEdit-homepage"
+                disabled={!user}
+                id="homepage"
+                name="homepage"
+                ref={(ref) => { this.homepage = ref; }}
+                defaultValue={user && user.homepage}
+                type="url"
+              />
+              <p className="UserProfileEdit--help">
+                {i18n.gettext(`This URL will only be visible if you are a
+                  developer.`)}
+              </p>
+
+              <label className="UserProfileEdit--label" htmlFor="location">
+                {i18n.gettext('Location')}
+              </label>
+              <input
+                className="UserProfileEdit-location"
+                disabled={!user}
+                id="location"
+                name="location"
+                ref={(ref) => { this.location = ref; }}
+                defaultValue={user && user.location}
+              />
+
+              <label className="UserProfileEdit--label" htmlFor="occupation">
+                {i18n.gettext('Occupation')}
+              </label>
+              <input
+                className="UserProfileEdit-occupation"
+                disabled={!user}
+                id="occupation"
+                name="occupation"
+                ref={(ref) => { this.occupation = ref; }}
+                defaultValue={user && user.occupation}
+              />
+            </Card>
+
+            <Card
+              className="UserProfileEdit--Card"
+              header={i18n.gettext('Your Biography')}
+            >
+              <label className="UserProfileEdit--label" htmlFor="biography">
+                {i18n.gettext(`Introduce yourself to the community. This text
```

Comment:
"yourself" & "your"

Reply:
Right. It seems like a lot of content/switches for admins though. Maybe there is a better way to do that, or maybe not..

Reply:
I'll try to avoid pronouns as per https://design.firefox.com/photon/copy/writing-for-users.html#point-of-view. That should result in less switches.

Reply:
Maybe we should ask @devaneymoz and/or @MeridelW for a review of this copy?